### PR TITLE
Add field `can_manage_video_chats` to ChatMemberOwner

### PR DESCRIPTION
**Pull request by andrew000**
# Description

Add field `can_manage_video_chats` to `ChatMemberOwner`

- New feature (non-breaking change that adds functionality)

# How has this been tested?

No tests, just add field

## Test Configuration
- Operating system: Win10
- Python version: 3.9.*

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works as expected
- [x] New and existing unit tests pass locally with my changes
- [x] My changes generate no new warnings or errors
- [x] My changes are compatible with minimum requirements of the project
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by evgfilim1** [comment]
I guess it must be renamed with deprecation

image
https://core.telegram.org/bots/api#april-16-2022

File: aiogram/types/chat_member.py [hunk 1]
```diff
@@ -129,6 +129,7 @@ class ChatMemberOwner(ChatMember):
     can_edit_messages: base.Boolean = fields.ConstField(True)
     can_delete_messages: base.Boolean = fields.ConstField(True)
     can_manage_voice_chats: base.Boolean = fields.ConstField(True)
+    can_manage_video_chats: base.Boolean = fields.ConstField(True)
```
Comment (Olegt0rr) [inline aiogram/types/chat_member.py:132]: Don't forget to check other usage of previous name

**Review by Olegt0rr — CHANGES_REQUESTED** [review]

**Comment by andrew000** [comment]
TESTS:
image

**Comment by Olegt0rr** [comment]
> TESTS: image

this should be issue for tests

**Comment by andrew000** [comment]
> > TESTS: image
> 
> this should be issue for tests

Since all the tests are passed, is it possible to merge the pr?

**Comment by andrew000** [comment]
> > TESTS: image
> 
> this should be issue for tests

Since all the tests are passed, is it possible to merge PR?

**Review by Olegt0rr — APPROVED** [review]

**Pull request merged**